src/Frost.ts

```diff
@@ -312,6 +310,28 @@ export class Frost {
       })
   }
 
+  removeApiToken(token: string, tokenId: string): Promise<string> {
+    const options = {
+      method: Method.DEL,
+      headers: new Headers({
+        'Content-Type': 'application/json',
+        token
+      })
+    }
+
+    const request = fetch(`${this.host}${Path.TOKENS}/${tokenId}`, options)
+
+    return Promise.race([request, this.timeoutPromise()])
```

Comment:
I'm wondering if we could avoid the use of `.then` by `await`ing the result of `Promise.race`...? That way we'd keep the code consistent everywhere.

```
const response = await Promise.race....
// if the timeout "won" the race, it'd have thrown, so we wouldn't reach the following line of code.
if (response.ok) return await value.text()
throw await value.text()
```

Something like this?

Reply:
yes, it's possible, I will apply a refactor if you want...

Reply:
Yes please

Reply:
(But only on this case, if there's a big refactor involved to things outside the scope of this PR, let's leave it to a future PR)